token-metadata/program/src/assertions/collection.rs

```diff
@@ -1,11 +1,12 @@
 use solana_program::{account_info::AccountInfo, program_error::ProgramError, pubkey::Pubkey};
 
 use crate::{
+    constants::{EDITION, PREFIX},
```

Comment:
As a mention, I've generally seen `defs` used as the module that holds various constants and other definitions in a bunch of projects.